tests/factories/user_category_factories.py

```diff
@@ -0,0 +1,12 @@
+from tests.factories import *
+
+from sourcemash.models import UserCategory, Category
+from user_factories import UserFactory
+
+class UserCategoryFactory(SQLAlchemyModelFactory):
+    FACTORY_FOR = UserCategory
+    FACTORY_SESSION = db.session
+
+    user = factory.SubFactory(UserFactory)
+    category = Category("category")
```

Comment:
But it does not make sense for the UserCategory factory to make an itemWithCategory just to get a category.